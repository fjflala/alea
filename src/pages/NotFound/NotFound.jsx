/**
 * Module dependencies
 */
import React from 'react';

/**
 * UI Components
 */
import Card from '../../components/Card';

/**
 * Represents a Not Found view
 */
const NotFound = () => (
  <section className="not-found">
    <Card>
      <p>Sorry, page not found :(</p>
    </Card>
  </section>
);

/**
 * Expose component
 */
export default NotFound;
