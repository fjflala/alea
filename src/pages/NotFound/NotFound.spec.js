import React from 'react';
import renderer from 'react-test-renderer';
import NotFound from './NotFound';

describe('<NotFound />', () => {
  it('should correctly render', () => {
    const tree = renderer
      .create(<NotFound />)
      .toJSON();
    
    expect(tree).toMatchSnapshot();
  });
});