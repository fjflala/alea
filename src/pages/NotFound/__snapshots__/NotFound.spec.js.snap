// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> should correctly render 1`] = `
<section
  className="not-found"
>
  <div
    className="ui-card "
  >
    <p>
      Sorry, page not found :(
    </p>
  </div>
</section>
`;
